### PR TITLE
wthrr 1.2.0 (new formula)

### PR DESCRIPTION
**Pull request by ivaquero**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/w/wthrr.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    system bin/"wthrr", "-h"
+
+    require "pty"
+    PTY.spawn(bin/"wthrr", "--save") do |_r, _w, pid|
```
Comment (carlocab) [inline Formula/w/wthrr.rb:24]: Let's read the output of this to make sure we got something sensible.

Comment (ivaquero) [reply Formula/w/wthrr.rb:24]: No problem, I will continue the work in the evening.

**Comment by ivaquero** [comment]
Sorry, I made a mistake move

**Comment by carlocab** [comment]
No problem! Feel free to create a new PR.

**Comment by chenrui333** [comment]
#191593

**Pull request closed without merging**